docs/jwst/flatfield/reference_files.rst

```diff
@@ -295,10 +331,7 @@ The FAST_VARIATION table contains four columns:
 * wavelength: float 1-D array, values of wavelength
 * data: float 1-D array, flat field values for each wavelength
 
-
-The flat field values in this table are used to account for a wavelength-dependence on a much
-finer scale than given by the values in the SCI array.  There is a single row in this table, 
-as the same wavelength-dependent value is applied to all pixels.
-
-.. include:: ../includes/dq_def.rst
+The flat field values in this table are used to account for a wavelength-dependence on a
+much finer scale than given by the values in the SCI array.  There is a single row in
+this table, because the same wavelength-dependent value is applied to all pixels.
```

Comment:
No, the wavelength and data in that row are arrays.

Reply:
fixed in new version